README.md

```diff
@@ -87,10 +87,12 @@ Continue to the next section to install java.
 **Untested**: While this fork has improved `fish` shell support, it has not been tested by this maintainer. To install `jenv` for Fish according to the contributor's instructions:
 
 ```sh
-echo 'set PATH $HOME/.jenv/bin $PATH' >> ~/.config/fish/config.fish
+# The line below is not needed if you installed jenv with Homebrew
+echo 'fish_add_path $HOME/.jenv/bin' >> ~/.config/fish/config.fish
```

Comment:
Also, currently by default `fish_add_path` uses a universal `$fish_user_paths` variable - stored in the `~/.config/fish/fish_variables` file. While `fish_add_path` checks for duplicates, it might be worth to use the `--global` flag.